cmd/install/assets/hypershift_operator.go

```diff
@@ -404,17 +401,6 @@ type HyperShiftOperatorDeployment struct {
 	RegistryOverrides                       string
 }
 
-// String returns a string containing all enabled feature gates, formatted as "key1=value1,key2=value2,...".
-func featureGateString() string {
-	featureGates := make([]string, 0)
-	for feature := range featuregate.MutableGates.GetAll() {
-		featureGates = append(featureGates, fmt.Sprintf("%s=true", feature))
-	}
-
-	sort.Strings(featureGates)
-	return strings.Join(featureGates, ",")
-}
```

Comment:
Removed because there are no longer any references to this function with this change.